docs/l4-l7.rst

```diff
@@ -186,8 +192,12 @@ New partitions and applications are listed in the Partition and Application drop
 
    - On clicking :guilabel:`Submit`, the partition is created asynchronously in the background. Once the partition is created successfully, the app UI loads the latest data. For more details, refer to https://clouddocs.f5.com/f5-aci-servicecenter/latest/l4-l7.html#as3-async-task-processing
    
-   - For :guilabel:`service-discovery/http` template, You can select APIC Tenant, Application, Endpoint Group from the provided drop-downs to associate them with the BIG-IP Application’s pool. Whenever the APIC Endpoint Group will have a new endpoint attached or detached from it, the FAST application’s pool will also have the same set of endpoints in it on the BIG-IP.
-   
+   - For :guilabel:`service-discovery/http` template, You can select APIC Tenant, Application, Endpoint Group from the provided drop-downs to associate them with the BIG-IP Application’s pool. The VRF field is a read-only field and is retrieved from the APIC’s Endpoint Group itself. You can select Route Domain from the provided dropdown options to associate them with the Application's pool. Whenever the APIC Endpoint Group will have a new endpoint attached or detached from it, the FAST application’s pool will also have the same set of endpoints in it on the BIG-IP.
+
+   - Route Domain must be created on the BIG-IP since the L2-L3 Network Management Tab as well as AS3 configuration does not support creation of Route Domains.
```

Comment:
Either "Route Domains..." or "The Route Domain..."
... the L2-L3 Network Management tab, as well as the AS3 configuration, does not....